packages/contracts/contracts/CyberDropBase.sol

```diff
@@ -31,24 +31,19 @@ contract CyberDropBase is CyberTokenBase {
     returns (
       uint256 timeStart,
       uint256 timeEnd,
-      uint256 priceStart,
-      uint256 priceEnd,
-      uint256 stepDuration,
+      uint256 price,
       uint256 amountCap,
       uint256 shareCyber,
       address creator,
       uint256 minted
     )
   {
     LibDropStorage.Drop storage drop = LibDropStorage.layout().drops[_tokenId];
-    require(drop.priceStart != 0, 'DNE');
```

Comment:
same as above

Reply:
we can't use price  because some drops have a price of 0 , si i used 
```sol
require(drop.amountCap != 0, 'DNE');
```

Reply:
yeah that why above you replace the check by amountCap, similar logic can be apply here